src/main/frontend/components/FilmShow.js

```diff
@@ -30,7 +30,7 @@ const FilmShow = props => {
 
   useEffect(() => {
     fetchFilm()
-  }, [])
+  }, [newReview])
```

Comment:
Ideally, rather than re-kick off the original (which means, in turn, an additional) fetch request, we would take the new review from our fetch response body and manually concatenate it into the film's existing reviews right here on the frontend!

Reply:
This is especially applicable since I don't think we're actually using that additional piece of state to render anything to the page

Reply:
Finally, I would also ask that you hand in something as the initial state on line 12 -- even if it's just `null`!

Reply:
fixed